<!--
ゲームパッド操作サンプルのlaunch

rosserialを使ってROSとメカナムローバーを接続し、PCにゲームパッドを接続してから起動してください。

起動コマンド：roslaunch megarover_samples joycon.launch
 -->

<launch>
  <!-- joy node -->
  <node pkg="joy" type="joy_node" name="mecanumrover_joy" respawn="true" >
    <param name="dev" type="string" value="/dev/input/js0" /> <!-- デバイスファイルパスを正しく設定してください -->
    <param name="deadzone" value="0.1"/>
  </node>

  <!-- joycon -->
  <node pkg="mecanumrover_samples" type="joycon" name="joycon" respawn="true">
    <param name="axis_linear_x" value="1" />
    <param name="axis_linear_y" value="0" />
    <param name="axis_angular" value="2" />
    <param name="scale_linear" value="0.8" />
    <param name="scale_angular" value="1.0" />
    <param name="safety_button" value="7" />
  </node>
    

</launch>
